src/bundle/Resources/public/scss/ui/modules/sub-items-list/_table.view.item.scss

```diff
@@ -45,30 +24,13 @@
             left: 0;
             width: calculateRem(30px);
             line-height: 0;
-            margin-top: -0.5px; // Fixes borders alignment on Safari and Edge
```

Comment:
Note: it seems that we no longer need this. Checked on latest Safari and Edge on HD and Retina displays.